test/models/database/post/mock_post_data.dart

```diff
@@ -25,6 +27,30 @@ class MockPostFirestore {
     );
   }
 
+  static PostFirestore createUserPost(
+    UserIdFirestore userId,
+    GeoPoint location,
+  ) {
+    final point = GeoFirePoint(location.latitude, location.longitude);
+
+    return PostFirestore(
+      id: PostIdFirestore(
+        value: DateTime.now().microsecondsSinceEpoch.toString(),
+      ),
+      location: PostLocationFirestore(
+        geoPoint: location,
+        geohash: point.hash,
+      ),
+      data: PostData(
+        ownerId: userId,
+        title: "title",
+        description: "desciption",
+        publicationTime: Timestamp.now(),
```

Comment:
I just think that we may want to create the timestamp from a predefined value.
Like that its value doesn't change between runs and the tests operate on the same data.
```suggestion
        publicationTime: Timestamp.fromMicrosecondsSinceEpoch(1000000),
```

Reply:
Applied in f6c7890